src/main/java/frc/robot/utilities/pid/PIDFMotor.java

```diff
@@ -1,12 +1,22 @@
 package frc.robot.utilities.pid;
 
-import com.ctre.phoenix.motorcontrol.ControlMode;
+import edu.wpi.first.util.sendable.SendableBuilder;
 
 /**
  * An interface used for motors that have a configurable PID
  */
 public interface PIDFMotor extends PIDFConfigurable {
     void setCoefs(PIDFCoefs pidfCoefs);
 
-    void tunePIDF(ControlMode controlMode);
+    void set(double output);
+
+    double get();
+
+    @Override
+    default void initSendable(SendableBuilder builder) {
+        PIDFConfigurable.super.initSendable(builder);
+        builder.setSmartDashboardType("PIDFMotor");
+        builder.setSafeState(() -> set(0));
```

Comment:
what does this function do? I know it was used by CTRE just want you to check because it might work for the tuning

Reply:
ask other programming team heads